proto/sifnode/ethbridge/v1/types.proto

```diff
@@ -62,4 +62,9 @@ enum ClaimType {
 }
 
 // GenesisState for ethbridge
-message GenesisState { string crosschain_fee_receive_account = 1; }
+message GenesisState { 
+    string crosschain_fee_receive_account = 1; 
+    map<string, uint64> ethereum_lock_burn_sequence = 2;
```

Comment:
1. if ethereum_lock_burn_sequence not set before, then no data will be included in genesis, it doesn't generate any empty/default value
2. 0 is expected if no vallue exists, to avoid such confusion, we set the lockBurnSequence starting from 1, also in contract